org.eclipse.lemminx/src/test/java/org/eclipse/lemminx/services/extensions/ErrorParticipantLanguageServiceTest.java

```diff
@@ -114,34 +103,18 @@ static class ErrorParticipantLanguageService extends XMLLanguageService {
 		public ErrorParticipantLanguageService() {
 			super();
 
-			this.registerCodeActionParticipant(new ICodeActionParticipant() {
-				@Override
-				public void doCodeAction(ICodeActionRequest request, List<CodeAction> codeActions,
-						CancelChecker cancelChecker) {
-					throw new RuntimeException("This participant is broken");
-				}
-			});
-			this.registerCodeActionParticipant(new ICodeActionParticipant() {
-				@Override
-				public void doCodeAction(ICodeActionRequest request, List<CodeAction> codeActions,
-						CancelChecker cancelChecker) {
-					Diagnostic diagnostic = request.getDiagnostic();
-					codeActions.add(ca(diagnostic, te(0, 0, 0, 0, "a")));
-				}
-			});
+			this.registerCodeActionParticipant((request, codeActions, cancelChecker) -> {
+throw new RuntimeException("This participant is broken");
```

Comment:
I had the `lambdaExpression` clean up enabled. Looks like it sometimes struggles with getting indentation correct. cc @rgrunber

Reply:
(I fixed it, but essentially sometimes the second 3rd, 4th... lines of a generated lambda aren't indented correctly.

Reply:
I've noticed this as well. Seems to place regular spaces even when the file is using tabs, in my case.

Reply:
This is upstream, I'll file it.